webpack.config.js

```diff
@@ -0,0 +1,137 @@
+            //         ]
+            //     })
+            // },
+            {
+                test: /\.(scss|css)$/,
+                use: extractPlugin.extract({
+                    fallback: 'style-loader',
+                    use: [
+                        { loader: 'css-loader', options: { importLoaders: 1 } },
+                        { loader: 'sass-loader' }
+                    ]
+                })
+            },
+            // {
+            //     test: /\.html$/,
+            //     use: ['html-loader']
+            // },
+            {
+                test: /\.(png|jpg|gif)$/,
+                use: [{
+                    loader: 'file-loader',
+                    options: {
+                        name: '[name].[ext]',
+                        outputPath: 'img/',
+                        publicPath: 'img/'
+                    }
+                }]
+            },
+            {
+                test: /\.woff2?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
+                use: "url-loader"
+            },
+            {
+                test: /\.(ttf|eot|svg)(\?[\s\S]+)?$/,
+                use: 'file-loader'
+            },
+            { test: require.resolve('jquery'), loader: 'expose-loader?jQuery!expose-loader?$' }
+        ]
+    },
+    plugins: [
+        //When src requires are fixed, enbale these
+        new webpack.ProvidePlugin({
+            // $: "jquery",
+            // jQuery: "jquery",
+            // "window.jQuery": "jquery",
+            // Tether: "tether",
+            // "window.Tether": "tether",            
+            // jstree: "js-tree/dist/tree.js"
+        }),
+        extractPlugin,
+        //new HtmlWebpackPlugin({
+        //    template: 'src/index.html',
+         //   filename: 'index.html'
+        //}),
+        new CleanWebpackPlugin([outputPath+"/*"]),
+        new CopyWebpackPlugin([
+            { from: '*.html', to: './', context: 'src/'},
+            { from: '*.json', to: './', context: 'src/'},
+            { from: '*.txt', to: './' , context: 'src/'},
+            { from: '*.ico', to: './' , context: 'src/'},
+            { from: '*.png', to: './' , context: 'src/'},
+            { from: 'strings/*.json', to: './' , context: 'src/'},
+            { from: 'scripts/*.js', to: './' , context: 'src/'},
+            { from: 'css/**/*', to: './' , context: 'src/'},
+            { from: 'bower_components/**/*', to: './' , context: 'src/'},
+            { from: 'components/**/*', to: './' , context: 'src/'},
+            { from: 'dashboard/**/*', to: './' , context: 'src/'},
+            { from: 'devices/**/*', to: './' , context: 'src/'},
+            { from: 'home/**/*', to: './' , context: 'src/'},
+            { from: 'legacy/**/*', to: './' , context: 'src/'},
+            { from: 'list/**/*', to: './' , context: 'src/'},
+            { from: 'thirdparty/**/*', to: './' , context: 'src/'},
+          ])
+    ],
+    optimization: {
+        minimizer: [
```

Comment:
will this also minify stuff, or only uglify?

Reply:
uglify will also minify.